docs/develop/01_sdk/02_Cookbook/02_Manage a Web3 name/01_claim.md

```diff
@@ -0,0 +1,35 @@
+---
+id: web3name-claim
+title: Claim a Web3 name
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Example1 from '!!raw-loader!@site/code-examples/core_features/web3names/1_web3name.ts';
+
+Web3 names are user-friendly aliases for a KILT DID.
```

Comment:
The initial paragraph is a repetition of what we have in the concepts section. Is this ok or should we get rid of this content altogether?

Reply:
I would get rid of that and link to the concepts section for an introduction. That's a too much text for a cookbook anyway, I'd say

Reply:
what makes sense here is a short summary of how to claim, and repeating the example code here is fine I think (which is imported anyway, so no need to maintain twice).

Reply:
I think we agreed to not have links from the cookbook to the concepts, but only the other way around. So I removed the section altogether. https://github.com/KILTprotocol/docs/pull/99/commits/557d48f0a5eef4787295a9dccd7477371f0ceb77